src/components/list/_macro.njk

```diff
@@ -20,13 +20,21 @@
     {% else %}
         {% set listEl = 'ul' %}
     {% endif %}
-        <{{listEl}} {% if params.id %}id="{{ params.id }}"{% endif %} class="ons-list{% if listEl == 'p' %} ons-list--p{% endif %}{% if params.classes %} {{ params.classes -}}{% endif %}{% if params.variants %}{% if params.variants is not string %}{% for variant in params.variants %} ons-list--{{ variant }}{% endfor %}{% else %} ons-list--{{ params.variants }}{% endif %}{% endif %}{% if otherClasses %} {{ otherClasses -}}{% endif %}"{% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %} {{ attribute }}{% if value %}="{{ value }}"{% endif %}{% endfor %}{% endif %}>
-            {%- for item in params.itemsList  -%}
-                {% if listLength > 1 or listEl == 'ul' %}
+
```

Comment:
If we remove this one line space, what impact does it have on the snapshots?

Reply:
> If we remove this one line space, what impact does it have on the snapshots?

It fails again. It makes sense now, I'll just remove the extra line

Reply:
I wonder if it would have failed in relation to the old snapshot (pre updating it).